CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Updated grunt-eslint to `^13.0.0`.
 - Moved eslint config file to home directory.
 - Moved jQuery download to package.json.
+- Updated grunt-banner to `^0.4.0` and updates banner ascii art and copy.
+- Changed bower.json authors array field to single field author,
```

Comment:
Nice! Good catch. Fixed in https://github.com/cfpb/cfgov-refresh/commit/eb1a0ce70b80154fbc714bfb03284efbfd381937 and https://github.com/cfpb/cfgov-refresh/commit/109ca0963f1934bba2b97f549cc9044af089a6d3